src/wi_stuff.c

```diff
@@ -385,10 +387,60 @@ static int    num_lnames;
 
 static const char *exitpic, *enterpic;
 
+static interlevel_t interlevel_exiting, interlevel_entering;
+
 //
 // CODE
 //
 
+boolean CheckConditions(interlevelcond_t *conditions)
+{
+    boolean result = false;
+
+    for (int i = 0; i < array_size(conditions); ++i)
+    {
+        interlevelcond_t cond = conditions[i];
+
+        switch (cond.condition)
+        {
+            case AnimCondition_MapNumGreater:
+                result &= (wbs->last + 1 > cond.param);
```

Comment:
Doesn't `&=` with a `false` value always return `false`?

Reply:
Right, this function doesn't work 😄 So far only `wi_interlvl.c` works.